code/modules/projectiles/projectile.dm

```diff
@@ -67,10 +67,26 @@
 	qdel(src)
 
 /obj/item/projectile/proc/on_hit(atom/target, blocked = 0, hit_zone)
+	var/turf/target_loca = get_turf(target)
 	if(!isliving(target))
 		return 0
 	var/mob/living/L = target
 	if(blocked < 100) // not completely blocked
+		if(damage && L.blood_volume && damage_type == BRUTE)

```

Comment:
That's... what I was asking, but I guess I didn't make it clear then. Just add `>= whatever` after `damage` and follow with the logic. But I'm happy I can read then